lib/y2_installbase.pm

```diff
@@ -267,6 +267,11 @@ sub select_specific_patterns_by_iteration {
     # delete special 'all' and 'default' keys from the check
     delete $patterns{default};
     delete $patterns{all};
+
+    if (is_sle('>=15-sp3')) {
+        record_soft_failure('bsc#1189550 - Problem with scroll bar event in pattern selection');
```

Comment:
I would move record_info into `workaround_bsc1189550` function

Reply:
it would appear too many times, to many soft-failed boxes. We need to remember to delete here, is the disadvantage but it is more clear as it is not trivial to find a code to detect it, as it happens always for the last item when scrolling.

Reply:
ok